### PR TITLE
WIP: Straighten quotes and then smarten them with js

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Straighten quotes and then smarten them with js
- NOTE: there might be better ways to include the [js](https://smartquotes.js.org/), and it isn't passing all the lint checks

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the straighted quotes are curly

### CODE REVIEW

**Comment by anthonydillon** [comment]
@pmahnke Got the package working via NPM and webpack but there an issue. It mangles image template includes.

**Comment by anthonydillon** [comment]
Filed this issue upstream to see if they could include a feature which we could use to ignore lazyloaded images. https://github.com/kellym/smartquotes.js/issues/51

**Comment by pmahnke** [comment]
@anthonydillon seems like it ate the image template tags?

``````

**Comment by anthonydillon** [comment]
Yes, that was my comment above.

**Comment by anthonydillon** [comment]
I am disappointed with the responses upstream here. It makes me question whether we should use it if it takes this long to have fixes landed.

File: package.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "@canonical/cookie-policy": "2.1.0",
     "@canonical/global-nav": "2.4.3",
     "@canonical/latest-news": "1.0.3",
+    "smartquotes": "^2.3.1",
```
Comment (pmahnke) [inline package.json:80]:
```suggestion
    "smartquotes": "^2.3.2",
```

bump version to noscript escape version

**Comment by anthonydillon** [comment]
Superseded by https://github.com/canonical-web-and-design/ubuntu.com/pull/8892

**Pull request closed without merging**